tests/unit/cartography/data/jobs/test_statement.py

```diff
@@ -0,0 +1,14 @@
+from cartography.graph.statement import GraphStatement
```

Comment:
Test is only tangentially unrelated to the bug fix but I thought it'd be good to have a sanity check that the GraphStatement was actually getting created with what we expected.